### DISCUSSION TITLE
sample rate playing back raw data

### DISCUSSION THREAD

**Discussion by tab-zero** [coqui-ai/TTS#1086] [General Q&A]
I am able to convert text to speech, stream it over the wire into the browser. However, when I play it on browser, it needs appropriate sample rate. What sample rate should be put in?

**Comment by erogol** [comment]
most models are 22050 but you need to check model's config.json to be sure.

**Reply by drevicko** [reply]
@erogol for reference, where do we find a model's config.json?  I'm not seeing model-specific config files in the TTS repo..
My particular interest is YourTTS

**Reply by drevicko** [reply]
For YourTTS, the returned audio is in wav format, which includes metadata on sample rate. In my case 16000.

**Reply by erogol** [reply]
you can find the folder with 

```python 
from TTS.utils.generic_utils import get_user_data_dir
get_user_data_dir("tts")
```
